scripts/ci/get_modified_tests.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+# A script to generate a list of tests that have changed or added and create an
+# arguemnts file for sanitycheck to allow running those tests with --all
+
```

Comment:
thats what the script does. It looks at any changed files, then walks up the tree and looks for yaml files, if it finds any, it adds to the set.